lms/static/js/student_profile/views/learner_profile_factory.js

```diff
@@ -40,7 +40,9 @@
                 required: true,
                 editable: 'always',
                 showMessages: false,
-                title: gettext('edX learners can see my:'),
+                title: interpolate_text(
+                    gettext('{platform_name} learners can see my:'), {platform_name: options.platform_name}
```

Comment:
I think @andy-armstrong raises a good point re: i18n and partial sentences (this could be a problem for agglutinative languages like Finnish and Turkish, for example). If this is an easy fix, I would suggest something like "Profile that other {platform name} learners see:" (which isn't technically a complete sentence, but the phrase is self-contained).